### ISSUE TITLE
Checkboxes too big--- cover the entire width

### ISSUE THREAD

**Issue by anjlidaryani** [alexed1/LightningFlowComponents#129]
Hi,

I am displaying checkboxes in the child question. It is appearing two big. Covers the entire page width. Is there a way I can set their width? 

Thanks

**Comment by alexed1** [comment]
Not today. It would be a good enhancement.

**Comment by anjlidaryani** [comment]
yes I can try to add it on my own just need some starting point.  This is what I am trying to write in css class.

.THIS cb {
    
      width: 50px;
}

**Comment by anjlidaryani** [comment]
Hi,

I need to reduce the size of checkboxes urgently. I can try to do it myself , just need some guidance. Please let me know where can I add css code to reduce the checkbox size?

**Comment by anjlidaryani** [comment]
@alex I inspected the checkbox element in google chrome. I found where the issue is:

.slds-form-element_stacked:not([class*="slds-size"]), .slds-form_stacked .slds-form-element:not([class*="slds-size"]), .slds-form_horizontal .slds-form-element_stacked:not([class*="slds-size"])

         { /* width: 100%;  */
              flex-basis: 100%; }

When I unchecked the width:100%  in inspect element, it worked. However, when I refresh the page, those settings were not applied. Seems like we need to update somewhere in css file, to get it reflected always.

**Comment by anjlidaryani** [comment]
@alex Please let me know where does all the css lives for this Lightning bundle, so that I could atleast make changes on my own.
